src/sentry/notifications/notifications/rules.py

```diff
@@ -115,6 +121,14 @@ def get_context(self) -> MutableMapping[str, Any]:
                     "subtitle": get_performance_issue_alert_subtitle(self.event),
                 },
             )
+        if self.group.issue_category not in [GroupCategory.PERFORMANCE, GroupCategory.ERROR]:
```

Comment:
Like (imagine it's a better name) `GroupCategoriesThatHaveCustomStuff`?

Reply:
Yeah, something like that. Maybe it should be a map of category -> custom email template? Or I guess if it follows a common naming pattern it could just be a set of which categories have a custom email.